frontend/src/pages/ActivityReport/Pages/components/RecipientReviewSection.scss

```diff
@@ -1,3 +1,7 @@
 .smart-hub-review-section .public-DraftStyleDefault-block {
     margin: 0;
-}
\ No newline at end of file
+}
+
+.smart-hub-review-section svg {
+    margin-right: .5rem;
```

Comment:
this could be replaced with className="margin-right-1" on the FontAwesome component